applications/TrilinosApplication/custom_processes/trilinos_levelset_convection_process.h

```diff
@@ -334,6 +316,58 @@ class TrilinosLevelSetConvectionProcess
     ///@name Private Operations
     ///@{
 
+    void InitializeConvectionStrategy(BuilderSolverPointerType pBuilderAndSolver)
+    {
+        KRATOS_TRY
+
+        // Get auxiliary member variables from base class
+        auto& r_base_model_part = BaseType::mrBaseModelPart;
+        const auto& r_level_set_var = *BaseType::mpLevelSetVar;
+        const auto& r_convect_var = *BaseType::mpConvectVar;
+
+        // Check the nodal database of the current partition
+        if (r_base_model_part.NumberOfNodes() > 0){
+            VariableUtils().CheckVariableExists<Variable<double>>(r_level_set_var, r_base_model_part.Nodes());
+            VariableUtils().CheckVariableExists<Variable<array_1d<double,3>>>(r_convect_var, r_base_model_part.Nodes());
+        }
```

Comment:
```suggestion
        VariableUtils().CheckVariableExists<Variable<double>>(r_level_set_var, r_base_model_part.Nodes());
        VariableUtils().CheckVariableExists<Variable<array_1d<double,3>>>(r_convect_var, r_base_model_part.Nodes());
```

I would still remove it, is confusing and unnecessary